components/log-viewer-webui/server/src/fastify-v2/plugins/external/env.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable sort-keys */
+import env from "@fastify/env";
+
+
+declare module "fastify" {
+    export interface FastifyInstance {
+        config: {
+            PORT: number;
+            HOST: string;
+            CLP_DB_USER: string;
+            CLP_DB_PASS: string;
+
+            RATE_LIMIT_MAX: number;
```

Comment:
we should also add `RATE_LIMIT_MAX` to `.env`

@coderabbitai create an issue to track this